src/main/java/org/prebid/server/bidder/conversant/ConversantBidder.java

```diff
@@ -138,14 +139,26 @@ private static Imp modifyImp(Imp imp, ExtImpConversant impExt) {
         return imp.toBuilder()
                 .displaymanager(DISPLAY_MANAGER)
                 .displaymanagerver(DISPLAY_MANAGER_VER)
-                .bidfloor(impExt.getBidfloor())
+                .bidfloor(getBidFloor(imp, impExt))
                 .tagid(impExt.getTagId())
                 .secure(getSecure(imp, impExt))
                 .banner(modifyBanner(banner, impExt.getPosition()))
                 .video(video != null && banner == null ? modifyVideo(video, impExt) : video)
                 .build();
     }
 
+    private static BigDecimal getBidFloor(Imp imp, ExtImpConversant impExt) {
+        BigDecimal impBidFloor = imp.getBidfloor() == null ? new BigDecimal(0.00) : imp.getBidfloor();
```

Comment:
Can be final

Reply:
Actually we don't need to create zero variable, we can work with null variable too, by creating method like this
```java
 private static boolean isValidBidFloor(BigDecimal bidFloor) {
        return bidFloor != null && bidFloor.compareTo(BigDecimal.ZERO) > 0;
    }
```
And use two checks with this method in if statement, or even better ternary return